src/purifier.js

```diff
@@ -162,6 +162,11 @@ export const WHITELISTED_ATTRS_BY_TAGS = {
     'custom-validation-reporting',
     'target',
   ],
+  'input': [
+    'value',
+    'mask-output',
+    'mask',
+  ],
```

Comment:
`value` and `mask` should already be allowed: https://github.com/cure53/DOMPurify/blob/master/src/attrs.js

Try it out by adding a simple test case in test-purifier.js.

Reply:
`mask` looks like it's only allowed for svg, and this is for an input tag. I'll check the error messages to see what the issue with `value` was.

Reply:
DOMPurify doesn't use tag-specific whitelists for attributes. We should add a unit test for this change in test-purifier.js anyways.

Reply:
Done. PTAL

Reply:
Done. PTAL